content/iaed/1000-algoritmos.sheet.md

```diff
@@ -20,45 +20,381 @@ O código disponível nesta página foi retirado de slides dos docentes, bibliog
 
 ## Algoritmos
 
-Qualquer um dos sorts precisa destes `define`
+Os algoritmos abaixo vão usar as seguintes abstrações (em coerência com os algoritmos disponibilizados pela docência):
 
-`embed:assets/1000-start.c`
+```c
+typedef int Item; // caso queiramos ordenar inteiros
+
+#define key(A) A
+#define less(A, B) (key(A) < key(B))
+#define exch(A, B)  \
+    {               \
+        Item t = A; \
+        A = B;      \
+        B = t;      \
+    }
+#define compexch(A, B) \
+    if (less(B, A))    \
+    exch(A, B)
+```
+
+Mais ainda, de realçar que todos os algoritmos apresentados nesta secção estão explicados na secção respetiva dos resumos, indicada a par do respetivo código.
 
 ### Selection Sort
 
-`embed:assets/1000-selection-sort.c`
+[Explicação nos resumos](../algoritmos-elementares-ordenacao#selection-sort).
```

Comment:
Even though it works, I prefer for all links to be absolute (i.e. `/iead/algoritmos-elementares-ordenacao#selection-sort`)
Same for the other links below

Reply:
Fixed in the next commit